OpenSim/Tools/InverseKinematicsTool.cpp

```diff
@@ -418,6 +430,23 @@ bool InverseKinematicsTool::run()
             kinematicsReporter.getPositionStorage()->print(_outputMotionFileName);
         }
 
+		if (modelMarkerErrors) {
+			Array<string> labels("", 4);
+			labels[0] = "time";
+			labels[1] = "total squared error";
+			labels[2] = "marker error RMS";
+			labels[3] = "marker error max";
+
+			modelMarkerErrors->setColumnLabels(labels);
+			modelMarkerErrors->setName("Model Marker Errors from IK");
+
+			IO::makeDir(getResultsDir());
+			string errorFileName = TrialName + "_ik_model_marker_errors";
```

Comment:
I think you could remove `model_` from here (and from the `modelMarkerErrors` variable); the errors aren't any more associated with the model markers than with the experimental markers.